Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -650,12 +690,10 @@ lemma integrable_add_iff_of_nonpos {f g : α → ℝ} (h_meas : AEStronglyMeasur
   exact integrable_add_iff_of_nonneg h_meas.neg (hf.mono (fun _ ↦ neg_nonneg_of_nonpos))
     (hg.mono (fun _ ↦ neg_nonneg_of_nonpos))
 
-@[simp]
```

Comment:
Can you explain what is the reason behind the simp attribute removal on this lemma and the next one?

Reply:
They are now provable by simp, so can't be simp lemmas.